binaries/parallel_info.cc

```diff
@@ -0,0 +1,86 @@
+/**
+ * Copyright (c) 2016-present, Facebook, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "ATen/Parallel.h"
```

Comment:
Ed recently expanded torch.__config__.show() in https://github.com/pytorch/pytorch/pull/18579

It'd be nice to expose it to python as well for debugging, but I wonder what's the good name is (or just fold it into __config__.show)

Reply:
nice, I'll check it and might merge some functionality with ATen/Version.h

Reply:
I think config.show is more for build config and should be constant, but information about parallelism can change, e.g. depends on environment vars, or hardware.

I could add directly into `torch/__init__.py` through e.g. `torch._C.parallel_info()`

Reply:
and implement parallel_info() in ATen/Parallel